docs/targets/README.md

```diff
@@ -0,0 +1,36 @@
+# Welcome to the TriggerMesh Targets Documentation
+
+The Triggermesh Targets are still in production so this document will be evoloving rapidly over the next several months. So check back for updates on current Targets as well as the addition of new ones!
+
+![](../images/tmLogo.png)
+
+## Current Supported TriggerMesh Targets 
+
+[Twillio](./twilio.md) : Send SMS messages via [Twillio](https://www.twilio.com/)
+
+With the Twillio Target users can send event driven data via SMS text-messages.
+
+[AWS](./aws.md) : Post events to several [AWS](https://aws.amazon.com) services.
+
+With the AWS target TM console users can send event driven data to the following AWS services:
+
+[Lambda](https://aws.amazon.com/lambda/)
+[SNS](https://aws.amazon.com/sns/)
+[SQS](https://aws.amazon.com/sqs/)
+[Kinesis](https://aws.amazon.com/kinesis/)
+
+[Confluent](./confluent.md) : Post events to [Confluent Cloud](https://www.confluent.io/)
+
+With the Confluent Target users can send event driven data to specified Kafka Topics within a Conflunet Cloud 
```

Comment:
there is no such thing as "event driven data". You can write "with the confluent target users can send messages to a specified Kafka topic"

typo: conflunet